### PR TITLE
[RUIN-58] Enable saving to local device with multiple file formats at once

### PR DESCRIPTION
**Pull request by JackMao981**
# Description
Multiple exports work for all pdf, html, and excel files. Although there is a problem with their encoding as they don't seem to save properly (i.e. it is saved in the file system, but you can't open the files because their encoding is wrong), I can continue to fix that another time. 

# Testing
1. Checkout the branch
2. Go to export and click it
3. Try checking any combination of files and see if it shows the file saved successfully message.
4. Check if the files actually exist in the file system.

### CODE REVIEW

**Comment by CaseyJMay** [comment]
My computer is telling me failed to save successfully (on an emulator) no matter what combination I choose!  Eek!

OH, also, are the tooltip logos supposed to be on the far right? That might be a part of master I just haven't seen yet.

**Comment by 18chowdhary** [comment]
For me, xlsx and html bring up a message that the app could save successfully, though I can't seem to find the saved files anywhere...pdf export brings up a message that the app could not save successfully, though I _could_ find a saved file. I tested on a tablet. We can retest and confirm.

**Comment by 18chowdhary** [comment]
When I tried to save a report as a PDF, a message dialog popped up saying "Failed to save report". I could save to HTML and/or XLSX though! I tested on a tablet.

File: components/SaveToDevice.js [hunk 1]
```diff
@@ -111,34 +103,52 @@ componentDidMount() {
       // for andorid: externalDirectoryPath: /storage/emulated/0/Android/data/com.ruina/files
       // for android: externalStorageDirectoryPath: /storage/emulated/0
       // for ios: DocumentDirectoryPath: /var/mobile/Containers/Data/Application/12F7361A-BC3E-42C9-B81E-FBBBF7BA3E2C/Documents
-      const path_ios = RNFS.DocumentDirectoryPath + '/' + this.state.filename + "." + format;
-      const path_android = RNFS.ExternalStorageDirectoryPath + '/' + this.state.filename + "." + format;
+      var path_ios = RNFS.DocumentDirectoryPath + '/' + this.state.filename;
+      var path_android = RNFS.ExternalStorageDirectoryPath + '/' + this.state.filename;
```
Comment (18chowdhary) [inline components/SaveToDevice.js:107]: Does the filename automatically include the format now?

**Review by 18chowdhary — CHANGES_REQUESTED** [review]

File: components/SaveToDevice.js (hunk 1 above)
Comment (JackMao981) [reply components/SaveToDevice.js:107]: Yes it does. This is not where the filneame is assigned though. In fact I believe these are not even used at all.

File: components/SaveToDevice.js [hunk 2]
```diff
@@ -2,13 +2,14 @@ import React, {Component} from 'react';
...
 import {TopNavigation, Card, CardHeader, Text, Button} from '@ui-kitten/components';
-import { Platform, StyleSheet, View, TextInput, PermissionsAndroid, Dimensions } from 'react-native';
+import { Platform, StyleSheet, View, TextInput, Dimensions } from 'react-native';
 import { MaterialDialog } from 'react-native-material-dialog';
 import { material } from "react-native-typography";
 import Pdf from 'react-native-pdf';
 import RNHTMLtoPDF from 'react-native-html-to-pdf';
 import JSONconverter from '../utils/jsonConverter';
 import backgroundSave from '../utils/backgroundSave';
+//import RNFetchBlob from 'rn-fetch-blob';
```
Comment (KristinAoki) [inline components/SaveToDevice.js:12]: Delete is not needed

**Review by KristinAoki — APPROVED** [review]
Please merge master before commiting

**Review by 18chowdhary — APPROVED** [review]

**Pull request closed without merging**